openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -597,6 +618,233 @@ Location: https://client.example.net/cb?
+
+Communication with the Interactive Authorization Endpoint MUST utilize TLS.
+
+The Authorization Server indicates support for interactive authorization by publishing the `interactive_authorization_endpoint` parameter in its Authorization Server Metadata. In this case, the Wallet SHOULD use this endpoint to obtain authorization.
+
+
+Note: This mechanism can only be used for interactions with the same Wallet that started the issuance process.
+
+The following Figure illustrates a flow using the Interactive Authorization Endpoint, where the Authorization Server requests presentation of a Credential from the Wallet before the issuance of another Credential to the Wallet.
+
+
+!---
+~~~ ascii-art
+ +-----------+            +----------------------+     +-------------------+    +--------------------+
+ |   Wallet  |            | Authorization Server |     | Credential Issuer |    | OpenID4VP Verifier |
+ +-----------+            +----------------------+     +-------------------+    +--------------------+
+       |                              |                        |                           |
+       |                              |                        |                           |
+       |----------------------------->|  (1) Interactive       |                           |
+       |                              |      Authorization     |                           |
+       |                              |      Request           |                           |
+       |                              |                        |                           |
+       |                              |                        |                           |
+       |<-----------------------------|  (2) Interactive       |                           |
+       |                              |      Authorization     |                           |
+       |                              |      Response          |                           |
+       |                              |      (presentation     |                           |
+       |                              |      request,          |                           |
+       |                              |      auth_session)     |                           |
+       |                              |                        |                           |
+       |---------------------------------------------------------------------------------->|  (3) OpenID4VP direct_post to Response URI
+       |                              |                        |                           |
+       |<----------------------------------------------------------------------------------|  (4) (redirect_uri)
```

Comment:
Are we assuming this redirect is to be treated opaquely here e.g the wallet shouldn't add any request parameters etc to it?

Reply:
Added a note to say this in the text.